src/main/resources/app_config.yml

```diff
@@ -47,4 +47,9 @@ smtpConfig:
   port: 25
   from: no-reply@kainos.com
 
-domain: http://localhost:8888
\ No newline at end of file
+domain: http://localhost:8888
```

Comment:
Maybe as we change app config, we might change it to recruitment-helper.kainos.com as well?